com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -4498,6 +4600,12 @@ void RenderDebug(HDCamera hdCamera, CommandBuffer cmd, CullingResults cullResult
                     PushColorPickerDebugTexture(cmd, hdCamera, m_IntermediateAfterPostProcessBuffer);
                 }
 
+                if (debugParams.exposureDebugEnabled)
```

Comment:
I haven't seen this in the rendergraph version. Could you add it?
(in HDRenderPipeline.Debug.cs)

Reply:
Do we have post process support for Rendergraph yet? I was under the impression we didn't have that yet. Can do tho

Reply:
Just the basic skeleton so indeed you won't be able to the get proper texture to send to the function I guess.
Can you add the call at the right place commented with a // TODO RENDERGRAPH above to at least make sure we don't forget?

Reply:
Done!